dockerfiles/nginx/proxito.conf.template

```diff
@@ -1,7 +1,7 @@
 # Proxito
 server {
-    listen 80 default_server;
-    server_name $NGINX_PROXITO_SERVER_NAME;
+    listen 8080;
```

Comment:
Regarding the readme change below, it's not clear to me why proxito is moving to 8080 here. Is wrangler proxying to port 8080 now?

Reply:
We are moving El Proxito away from port 80. This is because Wrangler is the one listening to 80 now, since it's the first application handling the request at `*.devthedocs.org` URLs (documentation URLs).

Once the request is received by Wrangler, it executes our `.js` script that _performs another request behind the scenes_ (using `fetch`) to El Proxito (NGINX container in port 8080[^1]) and gets the "original response". Then, it modifies that response to inject the HTML tags required.

This is exactly how CF works in production.

Does this make sense?

[^1]: 8080 is just a random/standard port.

Reply:
Well I understood the topology of wrangler mimicking Cloudflare in front of all requests, but it's still not clear what exactly is configured to forward to port 8080.

Is this wrangler configuration somewhere? Or are you describing something special in our code to forward to this container/port?

Reply:
https://github.com/readthedocs/common/pull/192 might make it more clear?

Reply:
Ah, yeah, this PR does not show how the `wrangler` command is executed. The PR that Eric pointed out is the one to take a look and I think it will make it more clear 👍🏼 . Let me know if this is still confusing.